### PR TITLE
docs: Add enable services and installation steps for TRACING.md

### LINKED ISSUE #679: Add enable services steps and installation step for the configuration file for tracing

### PR DESCRIPTION
**Pull request by GabyCT**
This adds the instruction of enabling the services and the installation of the
configuration file used for tracing.

Fixes #679

Signed-off-by: Gabriela Cervantes <gabriela.cervantes.tellez@intel.com>

### CODE REVIEW

File: TRACING.md [hunk 1]
```diff
@@ -169,6 +169,10 @@ following:
      - The `jaeger-client-socat-redirector.service` systemd service
        (see the [Guest to Host communication using VSOCK](#guest-to-host-communication-using-vsock) section).
        This service should be placed at `/usr/lib/systemd/system/` of the rootfs.
+       It is necessary to perform the following command on the service:
+       ```
+       $ systemctl enable jaeger-client-socat-redirector.service
```
Comment (jodh-intel) [inline TRACING.md]: Hi @GabyCT - can you make it clear these commands needs to be run from within the rootfs context. Something like:

```sh
$ sudo -E chroot "$rootfs" systemctl enable jaeger-client-socat-redirector.service
```

File: TRACING.md [hunk 2]
```diff
@@ -179,10 +183,18 @@ following:
...
+       ```
+       $ systemctl enable kata-journald-host-redirect.service
+       ```
 
      - The `kata-redirect-agent-output-to-journal.conf` systemd snippet
        (see the [Guest to Host communication using VSOCK](#guest-to-host-communication-using-vsock) section).
        This configuration file should be placed at `/etc/systemd/system/kata-agent.service.d/` of the rootfs.
+       It is necessary to perform the following command:
+       ```
+       $ install -o root -g root -m 0440 kata-redirect-agent-output-to-journal.conf
```
Comment (jodh-intel) [inline TRACING.md]: Isn't this command missing a destination directory?

**Comment by GabyCT** [comment]
@jodh-intel changes applied, thanks for the feedback :)

**Pull request merged**